sklego/preprocessing.py

```diff
@@ -0,0 +1,48 @@
+import pandas as pd
+from sklearn.base import BaseEstimator, TransformerMixin
+from sklearn.utils.validation import check_is_fitted
+
+
+class PandasTypeSelector(BaseEstimator, TransformerMixin):
+    """
+    Select columns in a pandas dataframe based on their dtype
+
+    :param include: types to be included in the dataframe
+    :param exclude: types to be exluded in the dataframe
+    """
+    def __init__(self, include, exclude=[]):
```

Comment:
could you turn the empty list into something that is not mutable? 

also, should `include` also not be optional?

Reply:
Good point, changed the defaults to None